src/components/CurriculumComponents/CurriculumVisualiserFilters/CurriculumVisualiserFiltersMobile.tsx

```diff
@@ -105,43 +179,60 @@ function StickyBit({
           >
-            <ButtonGroup
-              aria-label="Select a year group"
-              $overflowX={"auto"}
-              $overflowY={"hidden"}
-              $pb={8}
-            >
-              {yearOptions.map((yearOption) => (
-                <Box key={yearOption} $pt={8} $ml={5}>
-                  <Button
-                    variant="brush"
-                    aria-label={`Year ${yearOption}`}
-                    background={
-                      selectedYear === yearOption ? "black" : "grey20"
-                    }
-                    isCurrent={yearOption === selectedYear}
-                    key={yearOption}
-                    label={getYearGroupTitle(yearData, yearOption)}
-                    onClick={() => {
-                      onSelectYear(yearOption);
-                      trackSelectYear(yearOption);
+            <ScrollableWrapper>
+              <StyledButtonGroup aria-label="Select a year group">
+                {yearOptions.map((yearOption) => (
+                  <Box key={yearOption} $pt={8} $ml={5}>
+                    <FocusIndicator
+                      data-testid="year-group-focus-indicator"
+                      $display={"inline-block"}
+                      $mb="space-between-ssx"
+                      $mr="space-between-ssx"
+                      $background={
+                        isSelectedYear(yearOption) ? "black" : "white"
+                      }
+                      $color={isSelectedYear(yearOption) ? "white" : "black"}
+                      $borderRadius={"border-radius-s"}
+                      $font="heading-7"
+                      disableMouseHover={isSelectedYear(yearOption)}
+                    >
+                      <StyledButton
+                        data-testid="year-group-filter-button"
+                        aria-pressed={isSelectedYear(yearOption)}
+                        onClick={() => {
+                          onSelectYear(yearOption);
+                          trackSelectYear(yearOption);
 
-                      // HACK: Scroll into view used also in Lesson Overview - prevents rerender
-                      document
-                        .getElementById(`year-${yearOption}`)
-                        ?.scrollIntoView({ behavior: "smooth" });
-                    }}
-                    data-testid="year-group-filter-button"
-                  />
-                </Box>
-              ))}
-            </ButtonGroup>
+                          const targetElement = document.getElementById(
+                            `year-${yearOption}`,
+                          );
+                          if (targetElement) {
+                            targetElement.scrollIntoView({
+                              behavior: "smooth",
+                            });
+                            const yearHeading = document.getElementById(
+                              `year-${yearOption}`,
+                            ) as HTMLElement;
```

Comment:
Casting this as an HTML element may not be the best approach, I think it'll probably throw an error if it expects that element may not exist and then the undefined will be cast as a HTMLElement. A conditional might be better, e.g. having a `if(yearHeading && instanceof yearHeading ===...`